src/endpoint.js

```diff
@@ -74,6 +77,23 @@ module.exports = class AbstractEndpoint {
     return this
   }
 
+  // Turn on auto-batching for this endpoint
+  enableAutoBatch (autoBatchDelay) {
```

Comment:
It would be nice to also call this `autoBatch` like in the client, so it's consistent. Similar to `language`, `debugging` etc.